.eslintrc.js

```diff
@@ -1,30 +1,54 @@
 module.exports = {
-  'extends': 'eslint:recommended',
-    'parserOptions': {
-      'sourceType': 'module',
-      'ecmaVersion': 2017
+  'extends': [
+    'airbnb-base/legacy',
```

Comment:
As @noahlange reasonably said here https://github.com/mozilla/nunjucks/pull/1052#issuecomment-358370406 taking Airbnb confit isn't a good idea if you want to use ES6+ features in Nunjucks. And since you do, taking an Airbnb config will result in a battle with it. Doesn't sound right.

Reply:
airbnb-base/legacy is basically the airbnb rules without all the es6 stuff. It seemed like a good fit, it was the closest fit to my own personal preferences.

Reply:
> my own personal preferences

That's the thing which always troubles JS developers. Our own preferences. And so we desecrate the unification. Nevertheless, the project is your, so are the rules.